src/main/java/org/sonarsource/sonarlint/ls/backend/BackendServiceFacade.java

```diff
@@ -193,6 +192,23 @@ private InitializeParams toInitParams(BackendInitParams initParams) {
     );
   }
 
+  @NotNull
+  EnumSet<BackendCapability> getBackendCapabilities(BackendInitParams initParams) {
+    var backendCapabilities = EnumSet.of(BackendCapability.SMART_NOTIFICATIONS, BackendCapability.PROJECT_SYNCHRONIZATION,
+      BackendCapability.EMBEDDED_SERVER, BackendCapability.SERVER_SENT_EVENTS, BackendCapability.DATAFLOW_BUG_DETECTION,
+      BackendCapability.FULL_SYNCHRONIZATION);
+    if (telemetry != null && telemetry.enabled()) {
+      backendCapabilities.add(BackendCapability.TELEMETRY);
+    }
+    if (shouldEnableMonitoring()) {
+      backendCapabilities.add(BackendCapability.MONITORING);
+    }
+    if (initParams.isEnableSecurityHotspots()) {
```

Comment:
In theory this condition should be tested as well as two above. But in reality it's always true for VSCode, so it should be fine.